backend/testing/r_and_d/evidence_categorisation/experiments/run_experiment.py

```diff
@@ -0,0 +1,191 @@
+"""
+Run a single evidence categorisation experiment.
+
+Usage:
+    python run_experiment.py --model gpt-5-mini --prompt variant_a --dataset run_child_obesity
+"""
+
+import asyncio
+import sys
+from pathlib import Path
+import json
+from datetime import datetime
+import pandas as pd
+
+# Setup paths
+SCRIPT_DIR = Path(__file__).resolve().parent
+PROJECT_DIR = SCRIPT_DIR.parent
+BACKEND_DIR = PROJECT_DIR.parents[2]
+
+# Add paths for imports
+sys.path.insert(0, str(PROJECT_DIR))
+sys.path.insert(0, str(BACKEND_DIR))
```

Comment:
This looks a bit painful - maybe define BACKEND_DIR same way as we've defined TESTING_DIR and import it from `app` or `testing` ? (see [here](https://github.com/nestauk/discovery_policy_atlas/blob/dev/backend/testing/__init__.py))